### PR TITLE
NMS-13247: Meridian Installation Documents Incorrect, and other doc fixes 

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ ] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ ] Have you made a comment in that issue which points back to this PR?
* [ ] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13247
* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13242
* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13039

### CODE REVIEW

**Review by dino2gnt — COMMENTED** [review]
The repository configuration steps are different between Horizon and Meridian as well, and should be updated. It is correct for Horizon, but for Meridian the repository configuration comes from Sales and contains per-customer credentials.  Simplest process to document would be "Configure the repository by copying the meridian repository file you received from Sales to the Minion server."

File: docs/modules/deployment/pages/system-scalability/time-series-storage/newts.adoc [hunk 1]
```diff
@@ -55,7 +55,7 @@ org.opennms.newts.config.ttl=31540000<6>
 org.opennms.newts.config.resource_shard=604800<7>
 ----
 
-<1> Associate time series data by the foreign ID instead of the database generated Node-ID
+<1> Associate time series data by the foreign ID instead of the database generated Node-ID.
```
Comment (mmahacek) [inline docs/modules/deployment/pages/system-scalability/time-series-storage/newts.adoc:58]: Why add a period to this list item and not the rest of them?

File: docs/modules/operation/pages/operation/newts/introduction.adoc [hunk 2]
```diff
@@ -2,4 +2,11 @@
 [[ga-opennms-operation-newts]]
 = Newts
 
-This section describes how to configure _{page-component-title}_ to use _Newts_ and how to use _{page-component-title}_ to monitor your Cassandra cluster.
+link:http://newts.io/[Newts] is a time-series data schema for link:http://cassandra.apache.org/[Apache Cassandra].
+It enables link:https://en.wikipedia.org/wiki/Scalability#Horizontal[horizontally scale] capabilities for your time series storage and is an alternative to link:http://www.opennms.org/wiki/JRobin[JRobin] and link:http://oss.oetiker.ch/rrdtool/[RRDtool].
```
Comment (mmahacek) [inline docs/modules/operation/pages/operation/newts/introduction.adoc:6]: Possibly [horizontal scaling] for link text. Also, may want to have JRobin link to a non-wiki site, to remove reliance on that site.

File: docs/modules/operation/pages/operation/newts/cassandra-newts-jmx.adoc [hunk 3]
```diff
@@ -3,114 +3,114 @@
...
 
 == Read and Write Latency
 
-[options="header, autowidth"]
+[options="header, %autowidth"]
 |===
 | Name                | Description
-| `ReadTotalLatency`  | Local read metrics.
-| `WriteTotalLatency` | Local write metrics.
+| `ReadTotalLatency`  | Local read metrics
```
Comment (mmahacek) [inline docs/modules/operation/pages/operation/newts/cassandra-newts-jmx.adoc:53]: Restore period for consistency with rest of file.

File: docs/modules/operation/pages/operation/newts/cassandra-newts-jmx.adoc [hunk 4]
```diff
@@ -3,114 +3,114 @@
...
+| `WriteTotalLatency` | Local write metrics
```
Comment (mmahacek) [inline docs/modules/operation/pages/operation/newts/cassandra-newts-jmx.adoc:54]: Restore period for consistency with rest of file.

File: docs/modules/operation/pages/operation/newts/cassandra-newts-jmx.adoc [hunk 5]
```diff
@@ -3,114 +3,114 @@
...
 |===
 
 == Pending
 
-[options="header, autowidth"]
+[options="header, %autowidth"]
 |===
 | Name                 | Description
-| `PendingCompactions` | Estimate of number of pending compactions for this column family
+| `PendingCompactions` | Estimated number of pending compactions for this column family
```
Comment (mmahacek) [inline docs/modules/operation/pages/operation/newts/cassandra-newts-jmx.adoc:105]: Add period for consistency with rest of file.

File: docs/modules/operation/pages/operation/newts/cassandra-newts-jmx.adoc [hunk 6]
```diff
@@ -3,114 +3,114 @@
...
 | `PendingFlushes`     | Estimated number of tasks pending for this column family
```
Comment (mmahacek) [inline docs/modules/operation/pages/operation/newts/cassandra-newts-jmx.adoc:106]: Add period for consistency with rest of file.

**Comment by garcia-sr** [comment]
> 
> 
> The repository configuration steps are different between Horizon and Meridian as well, and should be updated. It is correct for Horizon, but for Meridian the repository configuration comes from Sales and contains per-customer credentials. Simplest process to document would be "Configure the repository by copying the meridian repository file you received from Sales to the Minion server."

What Dino says here is true. Current documentation shows: 

```
dnf -y install https://yum.opennms.org/repofiles/opennms-repo-stable-rhel8.noarch.rpm
rpm --import https://yum.opennms.org/OPENNMS-GPG-KEY
```
The repository link should coincide with the customer's Meridian repository that they are given when they purchase a license.

File: docs/modules/deployment/pages/system-scalability/time-series-storage/newts.adoc (hunk 1 above)
Comment (Bonrob2) [reply docs/modules/deployment/pages/system-scalability/time-series-storage/newts.adoc:58]: Oversight. Technically, 1 and 2 should have periods, as they are full sentences with verbs, and then the others should not. However, I think that always looks messy, even though it's grammatically correct. Will have to check new style guide to see what NH style guide says. May revise in future. For now I'll add periods to all of them.

**Comment by Bonrob2** [comment]
@dino2gnt  and Mark, I have updated the content as per your feedback.

**Review by mmahacek — APPROVED** [review]

**Review by dino2gnt — APPROVED** [review]
LGTM!

**Review by indigo423 — APPROVED** [review]
Yes, well done, LGTM! 👍

**Pull request merged**